passw0rd/src/main/kotlin/com/virgilsecurity/passw0rd/client/HttpClientProtobuf.kt

```diff
@@ -53,7 +58,7 @@ import com.virgilsecurity.passw0rd.protobuf.build.Passw0rdProtos
 /**
  * HttpClientProtobuf class is an implementation of http client specifically for work with Protobuf.
  */
-class HttpClientProtobuf(val serviceBaseUrl: String = SERVICE_BASE_URL) {
+class HttpClientProtobuf(private val serviceBaseUrl: String = SERVICE_BASE_URL) {
```

Comment:
Maybe it's better to leave opportunity for developer to get serviceBaseUrl so she can view it anytime?

Reply:
```suggestion
class HttpClientProtobuf(val serviceBaseUrl: String = SERVICE_BASE_URL) {
```
Why not, good point :)